Kubernetes/windows/debug/enablewinflags/winserver2022.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: enable-win-flags
+  labels:
+    app: enable-win-flags
+spec:
+  selector:
+    matchLabels:
+      name: enable-win-flags
+  template:
+    metadata:
+      labels:
+        name: enable-win-flags
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\SYSTEM"
+      hostNetwork: true
+      containers:
+      - name: enable-win-flags
+        image: mcr.microsoft.com/dotnet/framework/samples:aspnetapp

```

Comment:
Curious if we could use this image instead:
`mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0`

https://github.com/microsoft/windows-host-process-containers-base-image#dockerfile-example

It runs on both WS2022 and WS2019 and it is a very small image size.

Reply:
Thanks David. I will try with this image.

Reply:
@daschott Tried suggested image, but got following error.

 Normal   Pulling    12s (x2 over 27s)  kubelet            Pulling image "mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0"
  Warning  Failed     12s (x2 over 27s)  kubelet            Failed to pull image **"mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0": rpc error: code = NotFound desc = failed to pull and unpack image "mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0": no match for platform in manifest: not found**
  Warning  Failed     12s (x2 over 27s)  kubelet            Error: ErrImagePull

Reply:
![image](https://user-images.githubusercontent.com/36676176/234993004-08cdc702-975e-4970-8121-a4767371c6d7.png)
strange it seemed to have pulled for me on fresh Win2022 AKS Node.... I guess you can leave it as-is.